root/etc/e-smith/templates/etc/dnsmasq.conf/55DomainRedirection

```diff
@@ -0,0 +1,20 @@
+#
+# 55DomainRedirection
+#
+# This is intended for private nameservers: if you have a nameserver on your network which deals with names of the form
+# xxx.internal.thekelleys.org.uk at 192.168.1.1 then giving  the server=/internal.thekelleys.org.uk/192.168.1.1  will
+# send all queries for internal machines to that nameserver, everything else will go to the servers in /etc/resolv.conf
+# The  special  server  address '#'means, "use the standard servers", so server=/google.com/1.2.3.4
+# server=/www.google.com/# will send queries for *.google.com to 1.2.3.4, except *www.google.com which will be for‐
+# warded as usual.
```

Comment:
Move this to the PR description.
Also bear in mind that is not intended only for private nameserver, but mostly.